src/components/Tables/PowerSummaryTable.js

```diff
@@ -17,91 +17,308 @@ function PowerSummaryTableToolTip({ title, statusColor }) {
     </Tooltip>
   );
 }
+
 function PowerSummaryTable({
-  title, data, total, percent,
+  title, data = [], total = 0, percent = 0, deviceId = 'MPW1',
```

Comment:
We can't hardcode this, it should be reading this from the selected device option.

Reply:
Oh okay, right , this shall be done.

Reply:
This is still not resolved, please look.

Reply:
@ravikiranchollangi , this is a delicate feature, it would be time consuming to do these changes, I shall do it, but can we push it for MPW1 for this release for time being? We have one device for now, so it works on itself. 

The feature works now and I connected with @chernyee yesterday offline and he gave a thumbs up to the front end changes, just this hardcoding thing shall be taken care of in some time. 

@ravikiranchollangi we can put this feature now is what I feel, please do let me know what you think and please merge it if everything looks ok. 

Feature: tested and verified it works.